src/SingleReport/MultiTouch.cpp

```diff
@@ -0,0 +1,246 @@
+/*
+Copyright (c) 2021 ilufang
+See the readme for credit to other people.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "MultiTouch.h"
+
+// HID Report identifiers
+#define _HID_REPORTID_TOUCH 0x01
+#define _HID_REPORTID_FEATURE 0x05
+
+// First part of the descriptor. It appears only once
+static const uint8_t _hidReportDescriptorTouchscreen_1[] PROGMEM = {
```

Comment:
Suggested name: `_hidReportDescriptorMultiTouch`, to keep in sync with the other names. Or maybe you could rename multitouch to touchscreen? I guess that is a much better name and reflex the suggested API changed better (tap for example).

Reply:
Everything else renamed to Touchscreen